src/test_all.py

```diff
@@ -33,6 +35,22 @@
 def assert_language_name(name):
     assert name in language_names, f"Did not find `{name}` among iso639 language names: {language_names}"
 
+def skewness(result):
+    value,counts = np.unique(result, return_counts=True)
+    norm_counts = counts / counts.sum()
+    entropy=-(norm_counts * np.log(norm_counts)/np.log(len(value))).sum()
```

Comment:
If we choose a single base for all the tasks, we can't pick a threshold because the upper bound will increase with the number of classes. That's why I used the number of classes as a base here.